src/devices/LidarLiteV3/MeasurementRepetitionMode.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.TimeOfFlight.Models.LidarLiteV3
+{
+    /// <summary>
+    /// Measurement repetition modes
+    /// </summary>
+    public enum MeasurementRepetitionMode
```

Comment:
I'd remove the `Mode` prefix since it doesn't make name any clearer/more specific

Reply:
fixed